src/ci/docker/scripts/rfl-build.sh

```diff
@@ -28,7 +28,7 @@ rm -rf linux || true
 # Download Linux at a specific commit
 mkdir -p linux
 git -C linux init
-git -C linux remote add origin https://github.com/Rust-for-Linux/linux.git
+git -C linux remote add origin https://github.com/Darksonn/linux.git
```

Comment:
this needs a cfg(bootstrap) to get bumped on release

Reply:
I don't follow, what does the Linux kernel have to do with bootstraping?

I think I did everything mentioned at https://rustc-dev-guide.rust-lang.org/tests/rust-for-linux.html.

Reply:
Oops, I missed that the maintainers have taken responsibility for flipping this back and don't need extra reminders from Release.

Reply:
I am confused too -- this seems fine.

When v6.14-rc3 releases in a couple weeks, which is the one I am planning to get the fix into, I will send a PR with that tag, and we will be in the normal situation again.

Reply:
> Oops, I missed that the maintainers have taken responsibility for flipping this back and don't need extra reminders from Release.

I don't think there is a technical requirement to flip it back. I mean, the idea is that we do so, but it is orthogonal to the release, no? It is just a CI test. For instance, in an ideal world, we could be testing several supported Linux versions (e.g. the latest LTS).